colmena/redis/queue.py

```diff
@@ -26,8 +27,8 @@ def make_queue_pairs(hostname: str, port: int = 6379, name='method',
                      keep_inputs: bool = True,
                      clean_slate: bool = True,
                      topics: Optional[Iterable[str]] = None,
-                     proxystore_name: Optional[str] = None,
-                     proxystore_threshold: Optional[int] = None)\
+                     proxystore_name: Optional[Union[str, Dict[str, str]]] = None,
```

Comment:
My revised proposal is if a single name is passed, we use that with all topics (including 'default'), and if a dict `{topic: name}` is passed, then we use the proxystore backend associated with any topics in the dict. If a topic is not in the dict, we assume it was intentional to not use proxystore for that topic.

Reply:
I like it!